[DOC] additional overview content for MirrorMaker 2.0 (#2843)
strimzi/strimzi-kafka-operator: documentation/modules/overview/con-configuration-points-mirrormaker.adoc

To set up MirrorMaker, a source and target (destination) Kafka cluster must be running.

> You can use {ProductName} with MirrorMaker 2.0, though the earlier version of MirrorMaker continues to be supported.

Reviewer: ```suggestion
You can use {ProductName} with MirrorMaker 2.0, although the earlier version of MirrorMaker continues to be supported.
```

* Source cluster configuration to consume data from the source cluster
* Target cluster configuration to output data to the target cluster

[discrete]
=== Cluster configuration

You configure a `KafkaMirrorMaker2` resource to define the Kafka Connect deployment, including the connection details of the source and target clusters,
and start running a set of MirrorMaker 2.0 connectors to make the connection.

> Topic configuration is automatically synchronized between source and target clusters according to the topics defined in the MirrorMaker2 custom resource. Configuration changes are propagated to remote topics so that new topics and partitions are detected and created.

Reviewer 2: "... in the KafkaMirrorMaker2 custom resource ..."

* Source cluster configuration to consume data from the source cluster
* Target cluster configuration to output data to the target cluster

[discrete]
=== Cluster configuration

> You configure a `KafkaMirrorMaker2` resource to define the Kafka Connect deployment, including the connection details of the source and target clusters,

Reviewer: ```suggestion
You configure a `KafkaMirrorMaker2` custom resource to define the Kafka Connect deployment, including the connection details of the source and target clusters,
```

> and start running a set of MirrorMaker 2.0 connectors to make the connection.

Reviewer: ```suggestion
and then run a set of MirrorMaker 2.0 connectors to make the connection.
```

> Topic configuration is automatically synchronized between source and target clusters according to the topics defined in the `KafkaMirrorMaker2` custom resource.

Reviewer: ```suggestion
Topic configuration is automatically synchronized between the source and target clusters according to the topics defined in the `KafkaMirrorMaker2` custom resource.
```

Configuration changes are propagated to remote topics so that new topics and partitions are detected and created.
Topic replication is defined using regular expression patterns to _whitelist_ or _blacklist_ topics.

> MirrorMaker 2.0 connectors and related internal topics help manage the transfer and synchronization of data between the clusters.

Reviewer: ```suggestion
The following MirrorMaker 2.0 connectors and related internal topics help manage the transfer and synchronization of data between the clusters.
```

> MirrorSourceConnector:: _MirrorSourceConnector_ creates remote topics from the originating cluster.

Reviewer: ```suggestion
MirrorSourceConnector:: A _MirrorSourceConnector_ creates remote topics from the source cluster.
```

> MirrorCheckpointConnector:: _MirrorCheckpointConnector_ tracks and maps offsets for specified consumer groups using an _offset sync_ topic and _checkpoint_ topic.

Reviewer: ```suggestion
MirrorCheckpointConnector:: A _MirrorCheckpointConnector_ tracks and maps offsets for specified consumer groups using an _offset sync_ topic and _checkpoint_ topic.
```

The offset sync topic maps the source and target offsets for replicated topic partitions from record metadata.
A checkpoint is emitted from each source cluster and replicated in the target cluster through the checkpoint topic.
The checkpoint topic maps the last committed offset in the source and target cluster for replicated topic partitions in each consumer group.

> MirrorHeartbeatConnector:: _MirrorHeartbeatConnector_ periodically checks connectivity between clusters.

Reviewer: ```suggestion
MirrorHeartbeatConnector:: A _MirrorHeartbeatConnector_ periodically checks connectivity between clusters.
```
